ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiActiveContractsService.scala

```diff
@@ -53,6 +53,7 @@ private[apiserver] final class ApiActiveContractsService private (
       )
       .via(logger.logErrorsOnStream)
       .via(StreamMetrics.countElements(metrics.daml.lapi.streams.acs))
+      .via(logger.logTermination)
```

Comment:
Why do we have to add this to each API stream manually? Wouldn't this be better placed somewhere around `StreamingServiceLifecycleManagement`?

Reply:
Hmm, yes. Putting it around `StreamingServiceLifecycleManagement` is a good idea. As to adding it to each stream manually, that probably should stay so that

- you can decide which stream to log and which not
- you can customize the message so that you know what is it that is being closed